backend/authentication/adminPasswordConfirm.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+const AWS = require('aws-sdk');
+const db = new AWS.DynamoDB.DocumentClient();
+const uuid = require('uuid');
+require('dotenv').config();
+
+async function adminPasswordConfirm(event, context, callback) {
+    try {
+        // Get the recovery code from db for this user
+        const recoveryCodes = await db.scan({
```

Comment:
Avoid using the scan. It's time-consuming, and a performance penalty. Use instead  `query`. 
In this case, for using the query you have to add an index to property `code`